tests/test_integration/test_endpoints_crud.py

```diff
@@ -209,3 +209,91 @@ def test_post_proteomics(client, tokens, session, data_fixtures):
         models.Proteomics.id == response.json["id"]
     ).one()
     assert proteomics.full_name == proteomics_request["full_name"]
+
+
+def test_batch_post_proteomics(client, tokens, session, data_fixtures):
+    n = 100
+    proteomics_request = {
+        "body": [
+            {
+                "sample_id": data_fixtures["sample"].id,
+                "identifier": "P12345",
+                "name": "AATM_RABIT",
+                "full_name": "Aspartate aminotransferase, mitochondrial",
```

Comment:
You could increase test confidence by varying the name for each of the objects to make sure that each insert has the right data. (That's easy to check one by one with the current query below, but if you update it to select multiple objects you might need to make sure the order is correct somehow, maybe order by id or just sort programmatically or something)